### PR TITLE
P2531R0 C++ Standard Library Issues

### LINKED ISSUE #5260: [2022-02 LWG Motion 1] P2531R0 C++ Standard Library Issues
[P2531R0](https://wg21.link/p2531r0)

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #5260

### CODE REVIEW

**Review by JohelEGP — CHANGES_REQUESTED** [review]

File: source/diagnostics.tex [hunk 1]
```diff
@@ -1046,11 +1046,13 @@
...
-If the argument \tcode{ev} corresponds to a POSIX \tcode{errno} value \tcode{posv}, the
-function shall return \tcode{error_condition(posv, generic_category())}.
-Otherwise, the function shall return \tcode{error_condition(ev,
-system_category())}. What constitutes correspondence for any given operating
-system is unspecified.
+If the argument \tcode{ev} is equal to 0,
+the function returns \tcode{error_condition(0, generic_category())}.
+Otherwise,
+if \tcode{ev} corresponds to a POSIX \tcode{errno} value \tcode{pxv},
+the function returns \tcode{error_condition(pxv, generic_category())}.
```
[the remark this answers is not in the document]
Comment (jensmaurer) [reply source/diagnostics.tex:1053]: Because I get a LaTeX overfull `\hbox` otherwise.

[the remark this answers is not in the document]
Comment (JohelEGP) [reply source/diagnostics.tex:1053]: So they are equivalent anyways? [ab45510](https://github.com/cplusplus/draft/pull/5288/commits/ab45510119f3749bb0b6900a94895e4a6fe929cb) changes it as the issue requests, just for [2b58d30](https://github.com/cplusplus/draft/pull/5288/commits/2b58d30a4240944758446463a6312732887542b4) to change it to something else.

Searching for `pxv` either in https://cplusplus.github.io/LWG/issue3598 or the sources in the `main` branch has no hits. Perhaps you meant to introduce it here?

[the remark this answers is not in the document]
Comment (jensmaurer) [reply source/diagnostics.tex:1053]: First of all, there was a duplicate commit title. This was just wrong; I've squashed it appropriately.

Second, this very phrase introduces posv / pxv and uses that term a few words later. It's never used again, so we can pick any name. It just happens that the edits in the LWG issue caused LaTeX mayhem, so renaming posv to something slightly shorter was the easiest option.

[the remark this answers is not in the document]
Comment (JohelEGP) [reply source/diagnostics.tex:1053]: > Second, this very phrase introduces posv / pxv and uses that term a few words later. [...]

That makes sense. I don't know this component, so it wasn't obvious to me it was being introduced. I simply assumed it was referring to an existing `errno` value. Had I looked up `posv`, I would've figured that out. Sorry for the noise.

**Comment by tkoeppe** [comment]
@jensmaurer: is there a commit for LWG 3616? Or is that superceded by some later issue?

**Comment by tkoeppe** [comment]
Ah, looks like this has already been handled in 8753efabbbfec8371a82de1af60337fa6a6b6dc2.

**Pull request merged**